tests/functional/api/groups/read_test.py

```diff
@@ -92,6 +95,7 @@ def test_it_returns_http_200_for_private_group_with_creator_authentication(
     ):
         user, _ = user_with_token
         group = factories.Group(creator=user)
+        db_session.add(GroupMembership(group_id=group.id, user_id=user.id))
```

Comment:
The test is `test_it_returns_http_200_for_private_group_with_creator_authentication`. Semantically this test is actually dependent both on `user` being `group`'s `creator` and on `user` being a member of `group` so I've maintained both facts.